web/packages/teleport/src/Console/stores/types.ts

```diff
@@ -25,6 +25,11 @@ interface DocumentBase {
   url: string;
   kind: 'terminal' | 'nodes' | 'blank';
   created: Date;
+  latency: {
```

Comment:
Hmm, the latency is tied to a particular terminal session, right? 
If so, we shouldn't add this field here, but rather to `DocumentSsh` which sits below.
 
Then, `ActionBar` should only render the item if if this is a terminal session.
Now you can see the indicator also on nodes screen, which I believe doesn't make much sense.
![image](https://github.com/gravitational/teleport/assets/20583051/3f389da4-2de2-4050-b676-fc5a6be518ed)


I believe that the more correct state would be like this ⬇️
`Console.tsx`:
```ts
<ActionBar
    onLogout={onLogout}
    latencyIndicator={
    activeDoc.kind === 'terminal'
      ? {
            isVisible: true,
            latency: activeDoc.latency,
          }
         : { isVisible: false }
       }
    />
```

`ActionBar.tsx`:
```ts
{props.latencyIndicator.isVisible && (
   <LatencyDiagnostic latency={props.latencyIndicator.latency} />
 )}
...

type Props = {
  latencyIndicator:
    | { isVisible: true; latency: DocumentSsh['latency'] }
    | {
        isVisible: false;
      };
  onLogout: VoidFunction;
};
```

```stores/types.ts```:
```ts
export interface DocumentSsh extends DocumentBase {
  status: 'connected' | 'disconnected';
  kind: 'terminal';
  sid?: string;
  mode?: ParticipantMode;
  serverId: string;
  login: string;
  latency:
    | {
        client: number;
        server: number;
      }
    | undefined;
}
```